__tests__/endpoints.test.js

```diff
@@ -41,6 +41,26 @@ describe("/api/articles/:article_id", () => {
 	});
 });
 
+describe("api/articles", () => {
+	test("GET: 200 send an array of article objects to the client", () => {
+		return request(app)
+			.get("/api/articles")
+			.expect(200)
+			.then(({ body: { articles } }) => {
+				expect(articles).toBeSorted({key : 'created_at', descending : true,});
+				articles.forEach((article) => {
+					expect(article).toHaveProperty("author");
+					expect(article).toHaveProperty("title");
+					expect(article).toHaveProperty("article_id");
+					expect(article).toHaveProperty("topic");
+					expect(article).toHaveProperty("created_at");
+					expect(article).toHaveProperty("votes");
+					expect(article).toHaveProperty("comment_count");
+				});
+			});
```

Comment:
You could refactor this test to use a flexible jest matcher such as toMatchObject instead